manim/utils/config_utils.py

```diff
@@ -122,6 +121,27 @@ def _parse_file_writer_config(config_parser, args):
         [fw_config["save_last_frame"], fw_config["from_animation_number"]]
     )
 
+    # Read in the log level
+    quiet = getattr(args, "quiet")
+    if not quiet and default.getboolean("quiet", None) is not None:
+        quiet = default.getboolean("quiet")
+    verbose = getattr(args, "verbose")
+    verbose = "CRITICAL" if verbose is None and quiet else verbose
+    verbose = default["verbose"] if verbose is None else verbose
```

Comment:
I'm not sure I can follow this logic easily. Also, it seems that one of the conditionals here is catching the case when `args` doesn't have a `verbose` attribute, which presumably happens when the user doesn't pass the `verbose` flag. This is not how we want to handle things.

The config system is meant to be used as follows: if the user passes a value through CLI flag, we use that value. If they don't, we use whatever value is in the library-wide config file `default.cfg`. So the logic should be something like this:

```
verbose = args.verbose if args.verbose is not None else default["verbose"]
```

which takes just one line. Does this make sense? Does this yield the same logic as what you wrote before?

Reply:
It does yield the same logic except if `verbose` is not passed in the CLI flags but if `quiet` is (or if `quiet` is in the `default.cfg`). If the flag `quiet` flag is removed I'll write it as you put it.